content/embeds/k8s-admission-webhook-cert.md

```diff
@@ -17,13 +17,17 @@
     CERT=`kubectl get secret admission-tls -o jsonpath='{.data.cert}'`
     ```
 
-3. Create a patch file for the Kubernetes validating webhook, replacing `<namespace>` with the namespace where the REC was installed.
-
-  The `webhook.yaml` example is in [redis-enterprise-k8s-docs/admissiion](https://github.com/RedisLabs/redis-enterprise-k8s-docs/tree/master/admission)
+3. Create a Kubernetes validating webhook, replacing `<namespace>` with the namespace where the REC was installed.
+   
+    > The `webhook.yaml` template can be found in [redis-enterprise-k8s-docs/admission](https://github.com/RedisLabs/redis-enterprise-k8s-docs/tree/master/admission)
```

Comment:
I'm not sure how `>` lines are rendered in this website and if it's something we'd actually want for this sentence. Maybe it's best to just omit the `>`.

Reply:
done